modules.json

```diff
@@ -17,11 +17,15 @@
                         "branch": "master",
                         "git_sha": "c8e35eb2055c099720a75538d1b8adb3fb5a464c"
                     },
-                    "minimap2/align": {
+                    "minimap2/index": {
                         "branch": "master",
                         "git_sha": "c8e35eb2055c099720a75538d1b8adb3fb5a464c"
                     },
-                    "minimap2/index": {
+                    "multiqc": {
+                        "branch": "master",
+                        "git_sha": "ee80d14721e76e2e079103b8dcd5d57129e584ba"
+                    },
+                    "quast": {
                         "branch": "master",
                         "git_sha": "c8e35eb2055c099720a75538d1b8adb3fb5a464c"
```

Comment:
shouldn't quast be removed if you modified it?

Reply:
yes I should do that